modules/lzma/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ if(CONFIG_LZMA)
   )
 
   target_include_directories(liblzma PRIVATE
-    configs/
+    ${LZMA_DIR}/configs/
```

Comment:
Why do we still have this file?
Please remove whole folder and references to it.

Reply:
In that case, we have compliance errors or multiple Kconfig LZMA definitions. Every module or ext lib has the same structure. I tried before to remove all folder.

Reply:
So just do not define LZMA twice.

Reply:
Look on hal_telink. It's already done here.

Reply:
thank you, let me try this way again.